eradiate/scenes/biosphere.py

```diff
@@ -281,6 +276,21 @@ class HomogeneousDiscreteCanopy(SceneElement):
         Number of leaves in the canopy. Used only to display information about
         the object.
 
+    ``leaf_positions`` (list[list[float]]):
+        Lists all leaf positions as three vectors in cartesian coordinates.
+        Default: [[0., 0., 0.]] m.
```

Comment:
I'd rather set no default here or add validators to check that `leaf_positions` and `leaf_orientations` are consistent.